src/tag-input/src/TagInput.js

```diff
@@ -158,7 +166,10 @@ class TagInput extends React.Component {
   handleKeyDown = event => {
     const { selectionEnd, value } = event.target
 
-    if (event.key === 'Enter') {
+    const key = GET_KEY_FOR_TAG_DELIMITER[this.props.tagSubmitKey]
+
+    if (event.key === key) {
+      console.log(event.key)
```

Comment:
remove `console.log`

Reply:
oof -- good catch! we should lint for these things 😬